### ISSUE TITLE
[5.x]: Error when adding entries that have no Entry URI Format when trying to add them via the New Button - Causes Database Exception - yiildb Exception SQLSTATE[40001]: Serialization failure: 1213 Deadlock found when trying to get lock;

### ISSUE THREAD

**Issue by terryupton** [craftcms/cms#16166]
### Description
When trying to create a new entry using the red + New Entry Button - we are faced with the following error;

![Image](https://github.com/user-attachments/assets/01565be9-5b0f-4087-a717-81fbcf263003)

```
Database Exception - yiildb Exception
SQLSTATE[40001]: Serialization failure: 1213 Deadlock found when trying to get lock; try restarting transaction
The SQL being executed was: INSERT INTO 'craft_entries_authors* ('entryld", 'authorld', 'sortOrder') VALUES (32966, 1, 1)
Error Info: Array
10-40001
[2]
= Deadlock found when trying to get lock: try restarting transaction
→ Caused by: PDOException
SQLSTATE[40001]: Serialization failure: 1213 Deadlock found when trying to get lock; try restarting transaction in /var/www/html/vendor/yiisoft/yii2/db/Command.php at line
```

![Image](https://github.com/user-attachments/assets/196c4206-a0ad-4d61-b737-c8802c0de41e)

This only happens for entries that have no Entry URI Format.
Entries with a Entry URI Format add and create a new entry perfectly fine. 

We are using some entries as re-usable code/components and so have no  Entry URI Format.

### Steps to reproduce

1. Click the New Entry Arrow and pick an entry that has no Entry URI Format.
2. Error screen.
3. Refresh/relaod the screen and then the new entry screen is presented. 

### Craft CMS version

Craft Pro 5.5.2

### PHP version

8.3.10

### Database type and version

MySQL 8.0.36

### Installed plugins and versions

-

**Comment by brandonkelly** [comment]
I’m not able to reproduce that. Can you try enabling [safe mode](https://craftcms.com/docs/5.x/reference/config/general.html#safemode) and see if you still get the error? If not, it’s likely being caused by a plugin or module.

**Comment by terryupton** [comment]
Thanks @brandonkelly - I have enabled safe mode and I don't get the error.
Any idea on how to debug and track it down to which plugin? I don't have any modules.

**Comment by terryupton** [comment]
Ah ignore that. I have found the culprit. It is the cloner plugin by Verbb, it was the first one I disabled as i thought it might be the most likely one. Would be great if cloning fields and entryTypes and Sections was made native :-) 
Any plans?

having disabled and re-enabled I am now no longer seeing the issue at all. So I assume it was the cloner plugin. But could be jsut because I disabled one and re-enabled and it cleared the error...? 🤷🏻‍♂️
